### PR TITLE
add coatnet model

### PR DESCRIPTION
**Pull request by dewabratapandu**
ada beberapa variasi coatnet, tapi yang kupakai coatnet_0 (yang standarnya) untuk saat ini. bisa jadi berubah nanti tergantung mana yang punya loss atau acc terbaik

### CODE REVIEW

File: earthvision/models/coatnet.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
+"""Inspired by https://github.com/chinhsuanwu/coatnet-pytorch"""
+import torch
+import torch.nn as nn
+from einops import rearrange
```
Comment (rubentea16) [inline earthvision/models/coatnet.py:4]: add library ke requirements.txt and environment.yml @dewabratapandu

Comment (dewabratapandu) [reply earthvision/models/coatnet.py:4]: done

**Review by rubentea16 — APPROVED** [review]
